portfolio/src/main/webapp/index.html

```diff
@@ -8,11 +8,16 @@
   </head>
   <body>
     <div id="content">
-      <h1>Aymar Portfolio</h1>
+      <h1 id="header">Aymar Portfolio</h1>
       <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
```

Comment:
nit: you can omit the `</p>` (https://google.github.io/styleguide/htmlcssguide.html#Optional_Tags)

Reply:
Thank you. I will remove optional tags.